### PR TITLE
Kumar342/registration update

### PR DESCRIPTION
**Pull request by kumar342**
Hey team,

I added my projects and blogs , take a pull request.

please give a review to  my task

### CODE REVIEW

File: content/participants/ttjc@KumarHemanth342/index.md [hunk 1]
```diff
@@ -36,23 +36,75 @@ It would be good if someone tells me how to do auth for apps.
...
+Node and Express
+
+_hosted link_
+https://github.com/kumar342/Node_project_0320
+http://18.188.18.78/
 
 ### blogs
 
-same as projects, I will keep updating my blogs as and when done.
+1.Performing CRUD operations using Node js
```
Comment (gkanishk) [inline content/participants/ttjc@KumarHemanth342/index.md:103]: Hey @kumar342 ,
Have you written your blogs if yes please mention the description and link

Comment (kumar342) [reply content/participants/ttjc@KumarHemanth342/index.md:103]: Yes gkanishk, I added description and link to the blogs. Please review it

Thank you.

File: content/participants/ttjc@KumarHemanth342/index.md [hunk 2]
```diff
@@ -36,23 +36,93 @@ It would be good if someone tells me how to do auth for apps.
 not a lot of projects are done. I will keep updating it with real links when done.
 My projects:
```
Comment (gkanishk) [inline content/participants/ttjc@KumarHemanth342/index.md:37]: Can you differentiate between hosted link and GitHub link in the following format:
_`description`_:
_`Github link`_:
_`hosted link`_:

for every project

File: content/participants/ttjc@KumarHemanth342/index.md [hunk 3]
```diff
@@ -36,23 +36,93 @@ It would be good if someone tells me how to do auth for apps.
...
+A simple React app which shows the list of all breeds from the dog.ceo API and also a detail page which shows a random image for the picked dog (from the list).
 
 _stack_
+React
 
 _hosted link_
+https://github.com/kumar342/random-dogs-image-list-using-react
+https://random-dogs-image.netlify.app/
+
+#### project:Login system
```
Comment (gkanishk) [inline content/participants/ttjc@KumarHemanth342/index.md:81]: Where are the links to this one?

File: content/participants/ttjc@KumarHemanth342/index.md [hunk 4]
```diff
@@ -36,23 +36,93 @@ It would be good if someone tells me how to do auth for apps.
...
+
+3.How to add storage or volume to remote server in aws
+
+_description_
+In this, will add volume to remote server of a particualr AWS EC2 instance
+
+_hosted link_
+https://medium.com/@hemanth2gundala/how-to-add-storage-or-volume-to-remote-server-in-aws-112a10c06877
 
 #### why I liked GraphQL over REST?
```
Comment (gkanishk) [inline content/participants/ttjc@KumarHemanth342/index.md:127]: Please remove this sample part.

File: content/participants/ttjc@KumarHemanth342/index.md [hunk 5]
```diff
@@ -36,23 +36,93 @@ It would be good if someone tells me how to do auth for apps.
...
 
 _description_
+An Application build with Node js and deployed by using AWS
 
 _stack_
+Node and Express
+
+_hosted link_
+https://github.com/kumar342/Node_project_0320
+http://18.188.18.78/
```
Comment (gkanishk) [inline content/participants/ttjc@KumarHemanth342/index.md:99]: https://18.188.18.78/ is throwing some JSON value please have a look and update

Comment (kumar342) [reply content/participants/ttjc@KumarHemanth342/index.md:113]: hi  @gkanishk, Yes this link   https://18.188.18.78/ is throwing some JSON value from the get api . That's what i did .

and also login system(project) have only github link only.

**Pull request closed without merging**